pkg/streamingpromql/benchmarks/comparison_test.go

```diff
@@ -113,11 +120,14 @@ func TestBothEnginesReturnSameResultsForBenchmarkQueries(t *testing.T) {
 
 			prometheusResult, prometheusClose := c.Run(ctx, t, start, end, interval, prometheusEngine, q)
 			mimirResult, mimirClose := c.Run(ctx, t, start, end, interval, mimirEngine, q)
+			mimirResultWithQueryPlanner, mimirCloseWithQueryPlanner := c.Run(ctx, t, start, end, interval, mimirEngineWithQueryPlanner, q)
```

Comment:
(nit) more consistent naming with the other results etc.

```suggestion
			mimirWithQueryPlannerResult, mimirWithQueryPlannerClose := c.Run(ctx, t, start, end, interval, mimirEngineWithQueryPlanner, q)
```

Reply:
Done in 857ca3e